src/lib/reducers/content.js

```diff
@@ -15,15 +15,25 @@ const defaultState = {
 }
 
 export const reducer = (state = defaultState, action) => {
-  if (
-    action.type === SET_CONTENT &&
-    state[action.location] !== action.content
-  ) {
-    return { ...state, [action.location]: action.content }
+  if (action.type === SET_CONTENT) {
+    if (!action.location) return state
+    const newState = Array.isArray(state[action.location])
+      ? state[action.location].slice()
+      : []
+    // avoid doublon
```

Comment:
Pourquoi avoid les doublons ?

Reply:
This part was already in this API, i judge it necessary to still be handled, do you think this part is useless here?

Reply:
Mmh I don't see why its there, I think it was more to fix the shortcomings of the `this.prev` method. I do not see why duplicates should be ignored.

```
<BarLeft>A</BarLeft>
<BarLeft>A</BarLeft>
<BarLeft>B</BarLeft>
```

If you unmount the 2nd and 3rd <BarLeft />, you will have the Bar displaying nothing.

Reply:
Now store handle correctly components update with content replacing in the Map, no need this check anymore